scripts/autotracking.lua

```diff
@@ -156,21 +156,30 @@ function updateSlab(segment)
     end
 end
 
+originalBahamut = ""
```

Comment:
This should go up with the other globals. Also it should default to a hex number?

Reply:
This one, I left as a null value until the first time the tail is picked up to ensure that it would a) have a semi-fresh value to compare against and b) wouldn't return any value until after it's actually propagated. This was done to prevent any potential misfires. If you'd prefer, I can set it to `false` and get the same effect.